provider/oci/images.go

```diff
@@ -251,31 +268,30 @@ func getImageType(img ociCore.Image) ImageType {
 	return ImageTypeGeneric
 }
 
-func NewInstanceImage(img ociCore.Image, compartmentID *string) (imgType InstanceImage, err error) {
-	var base corebase.Base
+// NewInstanceImage returns a populated InstanceImage from the ociCore.Image
+// struct returned by oci's API. Also, it returns the architecture that is
+// detected from the image OperatingSystemVersion and it's name.
```

Comment:
The 2nd sentence doesn't match the function return values.

s/it's/its/

Reply:
I just rephrased it